src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsresettlementpassportapi/resource/CaseAllocationResourceController.kt

```diff
@@ -141,4 +141,46 @@ class CaseAllocationResourceController(private val caseAllocationService: CaseAl
     @Parameter(required = true)
     staffId: Int,
   ) = caseAllocationService.getAllCaseAllocationByStaffId(staffId)
+
+  @GetMapping("/{prisonId}", produces = [MediaType.APPLICATION_JSON_VALUE])
```

Comment:
Minor one but I think this id in the path looks like it's the id of the worker not the prison. Maybe it would be better as a mandatory query param?